docs/manage/pricing.md

```diff
@@ -40,6 +40,12 @@ _See the calculation in [Wolfram Alpha](https://www.wolframalpha.com/input?i=1+m
 
 `5K requests/second * 86400 seconds/day * 30.5 days/month * 1KB/request` = **$1976/month**
 
+##### You process an average load of 5K RPS with 800 bytes objects. Momento Serverless Cache would cost you:
+
+`5K requests/second * 86400 seconds/day * 30.5 days/month * 1KB/request` = **$1976/month**
```

Comment:
NIT: can we change the numbers a bit to provide variety from above example? I suggest 3K requests/second